Content/calendar.css

```diff
@@ -0,0 +1,18 @@
+﻿#tableId
+{
+    width: 100%;
+}
+
+#tableId td, #tableId th {
```

Comment:
for css class names, the convention we use is to use `-`s to separate the words, so write this as 
`#table-id`. An even better way to write css is to use semantic class names. Using semantic class names means
writing your css to reflect the html element you intend to target. Therefore, as an example, you can write 
`#calendar-id` instead of `table-id`. Naming variables is an extremely important and one of the hardest aspects of software development.

Reply:
Also, if you are using two selectors, it's considered better to write them in two separate lines, like this:
```
#calendar-id td,
#calendar-id th
```